src/docbuilder/rustwide_builder.rs

```diff
@@ -108,7 +108,7 @@ impl RustwideBuilder {
 
         let mut targets_to_install = TARGETS
             .iter()
-            .map(|t| t.to_string())
+            .map(|t| (*t).to_string())
```

Comment:
This dereference is redundant, probably inserted by clippy.
```suggestion
            .map(|t| t.to_string())
```

Reply:
It's actually a performance thing, `&str` has a fast, specialized `to_string` implementation while `&&str` does not

Reply:
Oh interesting. Can you leave that as a comment and maybe a link to the documentation?

Reply:
Best I've got is the [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#inefficient_to_string), but it explains it fairly well. `&str` has it's own `ToString` impl, while `&&str` goes through `Display`, which is much slower